gremlin-javascript/src/main/javascript/gremlin-javascript/.eslintrc.js

```diff
@@ -31,9 +31,9 @@ module.exports = {
   extends: ['eslint:recommended', 'prettier'],
   plugins: ['prettier'],
   rules: {
-    'prettier/prettier': ['error'],
```

Comment:
The changes in this file allow line endings to be `CRLF`, right? Is that really necessary? I guess we'd prefer if only Unix style line endings were committed, but maybe I'm missing something here?

Reply:
There are two ways we could resolve this. Either allow for `CRLF` formatted files or convert the existing javascript files that use the `CRLF` format to `LF`.  The former felt like a simpler fix which is why I disabled the check for line separators.

If we want to keep it only Unix style, we could go ahead with converting all the JS files that are affected.

Reply:
I think we can leave this one just to move this along. If someone wants to go through and convert all the line endings, that can be done separately.

Reply:
Oh, I wasn't aware that there are already files with `CRLF`. Then I understand the change. Thanks for the explanation!